doc/lua_api.txt

```diff
@@ -2046,6 +2046,22 @@ Elements
 * If `auto_clip` is `true`, the background is clipped to the formspec size
   (`x` and `y` are used as offset values, `w` and `h` are ignored)
 
+### `background[<X>,<Y>;<W>,<H>;<texture name>;<auto_clip>;<middle>]`
+
+* 9-sliced background. See https://en.wikipedia.org/wiki/9-slice_scaling
+* Middle is a rect which defines the middle of the 9-slice.
+	* `x` - The middle will be x pixels from all sides.
+	* `x,y` - The middle will be x pixels from the horizontal and y from the vertical.
+	* `x,y,x2,y2` - The middle will start at x,y, and end at x2, y2. Negative x2 and y2 values
+		will be added to the width and height of the texture, allowing it to be used as the
+		distance from the far end.
+	* All numbers in middle are integers.
+* Use a background. Inventory rectangles are not drawn then.
```

Comment:
What does this mean? Why are they not drawn?

Reply:
This is copied from the old element's description, I'm not sure what it means

Reply:
Remove it. I don't see any issue there.
![grafik](https://user-images.githubusercontent.com/1497498/59867623-fcf40380-938e-11e9-8f37-6cb8c3b0d336.png)